packages/schematics/angular/ng-new/schema.json

```diff
@@ -131,6 +131,12 @@
       "type": "boolean",
       "default": false,
       "alias": "S"
+    },
+    "empty": {
```

Comment:
hmm, what do you think about reversing the logic, so that the options all represent positives? Otherwise it feels like it's added retroactively

- new option would be `createProject` or similar
- it defaults to true
- if you want an empty project you do `ng new --createProject=false` (or do we support `ng new --noCreateProject`?)

Reply:
I actually disagree with this. I think the default should be negative and the options "turn them on"
As a user the cognitive load of turning on "empty" via `--empty` is easier that the idea of "create project false" via `--noCreateProject".

This is the same approach for most CLI tools I've used.

deleting a directory: `rm -rf` where you're setting recursive and force to true
pushing a branch `git push origin some-branch -f` sets force to true

Taking this to the next level, I think a standardization of the provided boolean options for schematics and architect should be done at some point. It will provide consistency to users about how to use any of the available boolean options.

Reply:
What I'm saying is, imagine you didn't know what is the current behavior of Angular CLI.

There is a command that creates a new workspace, it can optionally also create a project.

Empty is the natural default state of the new workspace. The option `--createProject` also does a second step of populating the new workspace.

When I say negative, I mean that `--empty` is saying that there's something that should *not* be created.

Now, it so happens that for backwards compat, our users expect that `--createProject` should be the default since that matches what we do today.

Reply:
I certainly concede to that idea, but one of the principles on which the CLI was started (at least in my eyes) was to get started quickly by reducing the barrier to entry. With that being said, I think the idea of changing the default behavior of `ng new` to not create a project would be a mistake.

Also, I think this conversation could/should be moved over to the actual issue for historical purposes.